### PR TITLE
added more verbose submitter installation instructions.

### PR DESCRIPTION
**Pull request by andymanaws**
### What was the problem/requirement? (What/Why)
Customers were failing to understand how to setup the submitter.
Some versions of C4D were also causing issues due to missing libraries.

### What was the solution? (How)
Just updated the README file to elevate the instructions more verbosely.

### What is the impact of this change?
Customer should be able to setup C4d submitters.

### How was this change tested?
In consulation with service team.

### Was this change documented?
This is a doc change.

### Is this a breaking change?
NO
----

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,49 @@ This library requires:
...
+A missing library error will manifest in errors that can be visible from the **Python** section of the **Extensions > Console** UI. These typically look like:
+
+```
+PySide6/__init__.py: Unable to import Shiboken from  ...
+```
+
+To remedy these errors, you can switch to a later version of Cinema 4D which resolves the missing libraries, or you can manually add them specifically to the Cinema 4D python module, e.g.
+
+```
+& "C:\Program Files\Maxon Cinema 4D 2024\resource\modules\python\libs\win64\python.exe"-m ensurepip
```
Comment (karthikbekalp) [inline README.md]: I personally used these commands and I find these to be incredibly useful. Thanks for writing these! 

But I wonder if this should be in `DEVELOPMENT.md` instead or if we should just have all the instructions from that page over here? 
https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/blob/mainline/DEVELOPMENT.md

For example, in blender (https://github.com/aws-deadline/deadline-cloud-for-blender/blob/mainline/DEVELOPMENT.md) and other repos we have installation instructions for submitters in `DEVELOPMENT.md`.

**Review by karthikbekalp — COMMENTED** [review]
A little clarification regarding the location for installation instructions, but otherwise the PR looks great!

File: README.md (hunk 1 above)
Comment (andymanaws) [reply README.md]: Sure - its a little detailed to be on the main page.

Can we add a line in 'Getting Started' to say something like 'For more detailed installation of the submitter and adaptor, please see the development.md' ....

Comment (karthikbekalp) [reply README.md]: Yup that makes sense to me.

**Comment by crowecawcaw** [comment]
PR looks great! There are a couple checks are failing that are easy to fix:
1. The PR title or commit name needs to follow the Conventional Commit standard. You can add `docs: ` to the beginning of the PR title.
2. The commits need to be signed. There are instructions for signing the commits you've pushed here: https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/pull/46/checks?check_run_id=30059458571

**Pull request closed without merging**